packages/lib/src/vis/tiles/TiledHeatmap.tsx

```diff
@@ -0,0 +1,68 @@
+import { clamp, range } from 'lodash';
+
+import { getInterpolator } from '../heatmap/utils';
+import { useFrameRendering } from '../hooks';
+import { useAxisSystemContext } from '../shared/AxisSystemContext';
+import TiledHeatmapLayer from './TiledHeatmapLayer';
+import type { TilesApi } from './api';
+import { useDataPerPixel } from './hooks';
+import type { ColorMapAttributes } from './models';
+
+interface Props extends ColorMapAttributes {
```

Comment:
Maybe just `ColorMapProps` and `...colorMapProps` below? The term "attributes" already refers to HDF5 and HTML attributes.